### ISSUE TITLE
BUG - sev-3 - All - Get rid of SR hint text for VCL and vaccines

### ISSUE THREAD

**Issue by TKDickson** [department-of-veterans-affairs/va-mobile-app#8455]
## What happened?

On the home, profile, health, benefits and payments screens, the screen reader announces almost everything as "[feature you'd go to], link". (ex: appointments, link)

The two exceptions are the [veterans crisis line, and the vaccines nav button, both of which have pretty redundant hint text (see video)](https://drive.google.com/file/d/1kZhHskRvOkfcDUP9xPEs2wmEWOy5qxne/view?usp=sharing).

## Specs:

- Device:
- OS:
- User Account:
- Accessibility State:

## Bug Severity - BE SURE TO ADD THE SEVERITY LABEL
 
See [Bug Tracking](https://department-of-veterans-affairs.github.io/va-mobile-app/docs/QA#issue-severity) for details on severity levels

- Impact: High Low 
- Frequency: High Low 

- 3 - Low
- 2 - High
- 1 - Critical

## Screen shot(s) and additional information

<details>
  <summary>Full JSON response for services related to issue (expand/collapse)</summary>
  

</details>

## Ticket Checklist
- [ ] Steps to reproduce are defined
- [ ] Desired behavior is added
- [ ] Labels added (front-end, back-end, global, Health, relevant feature, accessibility, etc)
- [ ] Estimate of 1 added (for front-end tickets)
- [ ] Added either to the relevant feature epic (if found during new feature implementation), or the relevant team's bug epic ([Global](https://app.zenhub.com/workspaces/va-mobile-60f1a34998bc75000f2a489f/issues/gh/department-of-veterans-affairs/va-mobile-app/8302), [Health & Benefits](https://app.zenhub.com/workspaces/va-mobile-60f1a34998bc75000f2a489f/issues/gh/department-of-veterans-affairs/va-mobile-app/8303), [API](https://app.zenhub.com/workspaces/va-mobile-60f1a34998bc75000f2a489f/issues/gh/department-of-veterans-affairs/va-mobile-app/8304), [QART](https://app.zenhub.com/workspaces/va-mobile-60f1a34998bc75000f2a489f/issues/gh/department-of-veterans-affairs/va-mobile-app/8305))

**Comment by brea11y** [comment]
@TKDickson I agree. Let's remove the hint from both of these. I thought we removed them when we cleaned up the strings last year, but must have missed these two. Both should be links, but hints are definitely redundant and can be removed.

**Comment by bischoffa** [comment]
Adding Global Team label

**Comment by TKDickson** [comment]
Found pre-existing bug (#8740). This change is good to go.

**Comment by TKDickson** [comment]
No further testing in develop needed
